docs/UserGuide.adoc

```diff
@@ -57,9 +57,18 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 === Adding a guest: `add`
 
 Adds a guest to the guest list for the purpose of assigning a room +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG] -r ROOM_NUMBER
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG] r/ ROOM_NUMBER
 from/START_DATE to/END_DATE`
 
+The room number must be a 3-digit number from 001 to 100. For example, 074 is
+valid but 128 is not.
+
+The dates can be expressed in DD/MM/YY or DD/MM/YYYY.
+
+Currently, a guest can only make one booking with the hotel. This will
+subsequently be modified such that a guest can make an unlimited number of
+bookings, but provided that he only checks in one booking at once.
```

Comment:
not true, guest can have multiple bookings at the same time (just different rooms), so whether a guest is already in the checked-in guest list does not affect the check-in command